.NET/Microsoft.Recognizers.Definitions.Common/English/DateTimeDefinitions.cs

```diff
@@ -146,7 +150,7 @@ public static class DateTimeDefinitions
       public const string NightRegex = @"\b(mid)?night\b";
       public const string CommonDatePrefixRegex = @"^[\.]";
       public static readonly string LessThanOneHour = $@"(?<lth>(a\s+)?quarter|three quarter(s)?|half( an hour)?|{BaseDateTime.DeltaMinuteRegex}(\s+(minutes?|mins?))|{DeltaMinuteNumRegex}(\s+(minutes?|mins?)))";
-      public static readonly string WrittenTimeRegex = $@"(?<writtentime>{HourNumRegex}\s+({MinuteNumRegex}|(?<tens>twenty|thirty|fou?rty|fifty)\s+{MinuteNumRegex}))";
+      public static readonly string WrittenTimeRegex = $@"(?<writtentime>{HourNumRegex}\s+((?<tens>twenty|thirty|fou?rty|fifty)(\s*-?\s*))?{MinuteNumRegex}(\s+(minutes?|mins?))?)";
```

Comment:
Shouldn't MinuteNumRegex be re-written to actually capture all minute numbers in written form instead of this?

Reply:
ok, modified MinuteNumRegex. 
Also modified DeltaMinuteNumRegex to capture all minute numbers because certain patterns were not covered (added test cases).